src/main/java/step4/Winner.java

```diff
@@ -0,0 +1,34 @@
+package step4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winner {
+    private final List<Car> carList = new ArrayList<>();
+    private final Cars cars;
+    private final MaxLocation maxLocation = new MaxLocation();
```

Comment:
```suggestion
    private final Location maxLocation;
```
Location을 따로 나눠주셨군요 👍 
개인적으론 Location에 변수명만 바꿔서 사용해주어도 되지 않나 싶어요
너무 클래스가 많아지면 코드를 읽는 사람이 알아야할 컨텍스트가 많아지거든요ㅎㅎ